src/components/axes/numericAxis.ts

```diff
@@ -194,12 +194,51 @@ export module Axis {
       var labelGroupTransform = "translate(" + labelGroupTransformX + ", " + labelGroupTransformY + ")";
       this._tickLabelContainer.attr("transform", labelGroupTransform);
 
+      this._showAllTickMarks();
```

Comment:
It's neater as it encapsulates the semantics in the name. I prefer this when reading code.